src/udev/udevadm-trigger.c

```diff
@@ -25,6 +25,30 @@ static bool arg_verbose = false;
 static bool arg_dry_run = false;
 static bool arg_quiet = false;
 static bool arg_uuid = false;
+static bool arg_entry_match = false;
+static bool arg_entry_nomatch = false;
```

Comment:
having too bools for this is weird, i mean, setting both to false and both to true has the same effect, so they aren't really independent bools as they look like to be.

maybe use a "tristate" here, like we use it across the codebase at various places: an `int` that is either < 0 (for "uninitialized"/"doesn't-apply"/"unset"), == 0 (for "false"), > 0 (for "true").